linera-execution/src/execution_state_actor.rs

```diff
@@ -395,12 +397,20 @@ where
                     ExecutionError::UnauthorizedHttpRequest(url)
                 );
 
-                let response = Client::new()
+                #[cfg_attr(web, allow(unused_mut))]
+                let mut request = Client::new()
                     .request(request.method.into(), url)
                     .body(request.body)
-                    .headers(headers)
-                    .send()
-                    .await?;
+                    .headers(headers);
+                #[cfg(not(web))]
```

Comment:
…or use the same timeout in either case? Might be less surprising, too.

Reply:
We can't, `reqwest` doesn't support timeouts on Wasm :/